docs/content/preview/yugabyte-voyager/migrate/assess-migration.md

```diff
@@ -30,14 +30,17 @@ Voyager then generates a report that includes:
 - **Performance metrics:** Analyzes workload characteristics to recommend optimizations in YugabyteDB.
 - **Migration time estimate:** Provides an estimated time for data import into YugabyteDB based on experimental data.
 - **Unsupported query constructs:** Identifies SQL features and constructs not supported by YugabyteDB, such as advisory locks, system columns, and XML functions, and provides a list of queries containing these constructs.
+- **Unsupported PL/pgSQL objects:** Identifies SQL features and constructs that are not supported by YugabyteDB, such as advisory locks, system columns, and XML functions, within PL/pgSQL objects in the source schema. It reports the individual queries within these objects that are not supported, such as queries in the PL/pgSQL block for functions and procedures, or the select statements in views and materialized views that contain unsupported constructs.
 
 {{< warning title="Note" >}}
```

Comment:
I find calling a warning a note a bit contradictory. A more descriptive title would help. None of these bullets seem 'warning' worthy tbh. I think this could be a regular paragraph, with intro
"When running migration assessment, keep in mind the following:"